### ISSUE TITLE
Some system query options don't work with $expanded collections

### ISSUE THREAD

**Issue by kpko** [OData/WebApi#400]
Hi,

I want to use a query like http://mysite.com/odata/Families?$expand=Members($oderby=FirstName;$top=1), but this doesn't work at all. 

This should be supported as stated in the OData Spec regarding $expand:

```The set of expanded entities can be further refined through the application of expand options, expressed as a semicolon-separated list of system query options, enclosed in parentheses, see [OData‑URL].
Allowed system query options are $filter, $select, $orderby, $skip, $top, $count, $search, $expand, and $levels.```

http://docs.oasis-open.org/odata/odata/v4.0/errata02/os/complete/part1-protocol/odata-v4.0-errata02-os-part1-protocol-complete.html#_Toc406398298

$select works in my queries (/odata/Families?$expand=Members($select=FirstName), but $oderby and $top do not work. The requests don't run into errors, they simply return the collection unchanged.

**Comment by kpko** [comment]
Hi,

is this project still active or is OData/WebApi development happening at another repo? Not only that I got no answer on this one so far, I created an issue several weeks ago and got no answer on this one, too. (https://github.com/OData/WebApi/issues/356)

**Comment by VikingsFan** [comment]
Related https://github.com/OData/WebApi/issues/349 , those nested query options is not support yet

**Comment by congysu** [comment]
Related issue https://github.com/OData/WebApi/issues/126, https://github.com/OData/WebApi/issues/127, https://github.com/OData/WebApi/issues/349.

**Comment by MarkAPhillips** [comment]
What is the ETA for this feature? - it seems to have been hanging around on and off for over a year - is anyone proactively working on this - or does it constantly get pushed back?

**Comment by s-granet** [comment]
I agree, this would be really useful. It seems kind of weird that I can't pull the top x records from an expanded record set.

**Comment by VikingsFan** [comment]
@s-granet @MarkAPhillips @kpko I have a PR to get this feature work, would you pls take a look at it?
https://github.com/OData/WebApi/pull/569

**Comment by VikingsFan** [comment]
Merged 3f568901da0a4d3b7d7aaecb29f66e5ce7e14429 PR #569
